src/containers/Redemptions.jsx

```diff
@@ -62,35 +103,164 @@ class Redemptions extends React.Component {
     this.state = {
       allActivities: [],
       filteredActivities: [],
-      selectedStatus: 'All',
-      initialStatus: 'All',
+      societyRedemptions: [],
+      selectedStatus: ALL,
+      initialStatus: ALL,
```

Comment:
If `ALL` is a string, give it a more descriptive variable name.

Reply:
All could refer to either All activities or All redemptions. Any suggestions on what would be more descriptive?